src/Contracts/Form/index.js

```diff
@@ -122,6 +128,34 @@ const Form = props => {
   )
 }
 
+// NOTE copied from web3 v1.0 isAddress. Remove when we upgrade to web3 v1.0
+const isAddress = address => {
+  if (!/^(0x)?[0-9a-f]{40}$/i.test(address)) {
+    // check if it has the basic requirements of an address
+    return false
+  } else if (/^(0x)?[0-9a-f]{40}$/.test(address) || /^(0x)?[0-9A-F]{40}$/.test(address)) {
+    // If it's all small caps or all all caps, return true
+    return true
+  } else {
+    // Otherwise check each case
+    return isChecksumAddress(address)
+  }
+}
```

Comment:
I think there is a library for that else you can use this `const isAddr = addr => /^(0x)?[0-9a-f]{40}$/i.test(addr) || /^(0x)?[0-9a-f]{64}$/i.test(addr)`